static/app/components/workflowEngine/ui/textBadge.tsx

```diff
@@ -0,0 +1,16 @@
+import styled from '@emotion/styled';
+
+import {space} from 'sentry/styles/space';
+
+export const TextBadge = styled('span')`
```

Comment:
Is there a better name we could use here. Exporting something called TextBadge could be confusing to people looking for our Badge component. I assume its the Purple thing. ConditionBadge or something